dags/mantaray/run_mantaray_jobs.py

```diff
@@ -47,7 +47,9 @@
       catchup=False,
   ) as dag:
     for workload_file_name in workload_file_name_list:
-      run_workload = mantaray.run_workload(
+      run_workload = mantaray.run_workload.override(
+          task_id=workload_file_name[:-3]
```

Comment:
nit: I saw one task with a "." at the end. Is that because the workload file name extension may not be 3 characters? Is it possible to use `split` to split out the extension cleanly?

Reply:
Good suggestion, updated to `split` solution